doc/integrator/requirements.rst

```diff
@@ -55,6 +55,17 @@ default is: ``http://localhost:8080/print-c2cgeoportal-{instanceid}/pdf/``.
 And by default the 'webapps' folder is ``/srv/tomcat/tomcat1/webapps``,
 to change it set the ``PRINT_OUTPUT`` value in the makefile.
 
+For MapFish Print to function properly it is recommended to increase the default
+Java Heap Size for Tomcat (parameter ``-Xmx`` and ``-Xms``). On servers managed
+by Camptocamp check the size in the file ``/srv/tomcat/tomcat1/bin/setenv-local.sh``:
+
+  .. code:: make
+
+    export JAVA_XMX="1G"
+    export ADD_JAVA_OPTS="-Xms1G"
```

Comment:
This should be always done ? 
Do we relay need 1G, I think that 512M is enough...
Why do you add Xms ?